draft-ietf-scitt-architecture.md

```diff
@@ -126,63 +126,47 @@ Within the SCITT Architecture, a producer is known as an Issuer, and a consumer
 
 # Introduction
 
-This document describes a scalable and flexible, decentralized architecture to enhance auditability and accountability across various existing and emerging supply chains.
-It achieves this goal by enforcing the following complementary security guarantees:
-
-1. Statements made by Issuers about supply chain Artifacts must be identifiable, authentic, and non-repudiable
-1. Such Statements must be registered on a secure Append-only Log, enabling provenance and history to be independently and consistently audited
-1. Issuers can efficiently prove to any other party the Registration of their Signed Statements; verifying this proof ensures that the Issuer is consistent and non-equivocal when producing Signed Statements
+This document describes the scalable, flexible, and decentralized SCITT architecture.
+Its goal is to enhance auditability and accountability across supply chains.
+
+In supply chains, there are producers and consumers of items.
+Consumers may like to have information about the items that they consume, such as an item's provenance.
+SCITT provides a way for consumers to obtain this information in a way that is "transparent", that is, producers cannot lie about the information without it being detected.
+SCITT achieves this by having producers (also called Issuers) publish information in a Transparency Service, where consumers (also called Verifiers) can check the information.
+
+SCITT consists of the following roles: Issuers, Transparency Services, Verifiers, and Auditors.
+The process works as follows:
+
+1. Issuers make Statements about Artifacts they release, for example: "Issuer name", "Artifact type", "Artifact version".
```

Comment:
In my current mental model, the issuer is the entity that commits to the statement, not necessarily the entity that first made the statement... I don't know if this distinction is worth preserving.

Reply:
That's an interesting point, given that there are many derivative software components from different entities, I think it is important to consider (an Issuer can be the entity who made the software, someone who made a derivative, neither of those and someone who runs conformity/quality testing on software, etc).

Thanks for pointing this out, because I had not internalized this important distinction until now.

Reply:
Yes, I agree with @OR13 on this one: the only thing we can prove for certain is that the Issuer committed the statement to the TS (and even that's not certain if you want to get all Red Team on us, but we have to draw a line at some point so in the model we assume use of the Issuer Signing Key is foolproof).

As Orie says, they might simply be endorsing a statement made by someone else.
But more importantly it's absolutely not necessarily the case that that Issuer released the Artifact: they might be making observations about someone else's work. 

I'll make a suggestion to this effect

Reply:
agreeing with the points that the issuer is simply "owing" the statement being made about an artifact, not limited to a producer
- Producers of artifacts make statements about their artifacts (Wabbit Networks (issuer) produced some software)
- Auditors make statements about artifacts produced by others ("ScanAuditing" (issuer) attested to an artifact's release process )
- Security companies make statements about the current known quality of artifacts ("CogsWell Security" (issuer) found the net-montior (artifact) shipped by Wabbit Networks is now known to have a vulnerability and no longer meets X123 compliance requirements)
- Consumers make statements about the artifacts they consume, creating a history of their decisions (ACME Rockets (issuer) evaluated the net-monitor software on January 2nd 2024, found it met their security requirements and deployed it)